package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "sass": "1.32.5",
+    "synapse-react-client": "1.15.64"
```

Comment:
Will need to release a new version of SRC after merging https://github.com/Sage-Bionetworks/Synapse-React-Client/pull/883/files and bump here